pyatlas/unit_tests/test_atlas.py

```diff
@@ -0,0 +1,139 @@
+import unittest
+
+from pyatlas.atlas import Atlas
+from pyatlas import geometry
+from pyatlas.geometry import Rectangle
+
+
+class AtlasTest(unittest.TestCase):
```

Comment:
If those tests fail, are they build breaking?

Reply:
Yes, failing unit tests cause the build to abort with a diagnostic detailing the failed test and the cause of the failure. It's very similar to the JUnit setup we have in Java Atlas.